config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -796,14 +796,14 @@ presubmits:
 
 periodics:
 - interval: 30m
-  name: ci-kubernetes-e2e-gci-gce
+  name: ci-kubernetes-e2e-gce-cos-master-default
```

Comment:
@Rajalakshmi-Girish let's do a bulk change of the names at the end, let's focus on getting the rest of the changes right in this first set of PRs.

(changing names is hard as the names are muscle memory for folks and continuity is lost in testgrid etc)